### PR TITLE
Feat: Added Quiz result page for Quiz App

### PR DESCRIPTION
**Pull request by bmendhe23**
##Description
Feat:

1. Added Quiz Question pages for Quiz app.

##Screenshots
ss

##Devices Tested
[✔️] Desktop
[❌] Tablet
[❌] Mobile

### CODE REVIEW

File: quiz-question/question2.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </nav>
+    </header>
+    <main>
+        <div class="question-container">
+            <h2>I will be the Pirate King</h2>
+            <section class="quiz-info">
+                <div class="question-no"><strong>Question: </strong>2/5</div>
+                <div class="score"><strong>Score: </strong>10</div>
+            </section>
+            <section class="question-info">
```
Comment (bhavyaj12) [inline quiz-question/question2.html:45]: Nice use of semantic tags

File: quiz-question/question.css [hunk 2]
```diff
@@ -1,9 +1,80 @@
...
+.question-info .question {
+    text-align: justify;
+}
+
+.quiz-info {
+    display: flex;
+    justify-content: space-between;
+}
+
+.quiz-options li {
```
Comment (bhavyaj12) [inline quiz-question/question.css:36]: Good class names

File: quiz-question/question5.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                </li>
+                <li>
+                    <a href="#" class="nav-item">About us</a>
+                    <p class="active"></p>
+                </li>
+            </section>
+        </nav>
+    </header>
+    <main>
+        <div class="question-container">
```
Comment (bhavyaj12) [inline quiz-question/question5.html:39]: If possible you can add comments for readability then it will be perfect

**Review by bhavyaj12 — COMMENTED** [review]
Looks good to me! The colors can be better though make sure to check contrast between the green and the other colors

File: quiz-question/question2.html (hunk 1 above)
Comment (bmendhe23) [reply quiz-question/question2.html:45]: thank you @bhavyaj12

File: quiz-question/question.css (hunk 2 above)
Comment (bmendhe23) [reply quiz-question/question.css:36]: thank you @bhavyaj12

File: quiz-question/question5.html (hunk 3 above)
Comment (bmendhe23) [reply quiz-question/question5.html:39]: sure will do that

**Review by harshitpaliwal95 — APPROVED** [review]
- looks good
- clean ui

**Comment by bmendhe23** [comment]
> * looks good
> * clean ui

@harshitpaliwal95 thank you

**Pull request merged**